aicsimageio/readers/ome_zarr_reader.py

```diff
@@ -127,8 +127,14 @@ def _read_immediate(self) -> xr.DataArray:
 
     def _xarr_format(self, delayed: bool) -> xr.DataArray:
         image_data = self._zarr.load(str(self.current_scene_index))
-        axes = self._zarr.root_attrs["multiscales"][self.current_scene_index]["axes"]
-        dims = [sub["name"].upper() for sub in axes]
+
+        if "axes" in self._zarr.root_attrs["multiscales"][self.current_scene_index]:
```

Comment:
I wonder if this check would be cleaner or more readable by using dict `.get` and checking the result for None.

Reply:
Like this
`self._zarr.root_attrs["multiscales"][self.current_scene_index].get("axes")`

Reply:
added in [16bf109](https://github.com/AllenCellModeling/aicsimageio/pull/514/commits/16bf10994beb9f2df1df41c659eb5af390d633a5) with the slight adjustment ` if axes:` rather than `if axes is None:`